src/main/resources/mesosphere/marathon/api/v2/AppDefinition.json

```diff
@@ -140,7 +140,7 @@
         },
         "dependencies": {
             "items": {
-                "pattern": "^/?(([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])\\.)*([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])$",
+                "pattern": "^(\\/?((\\.{2})|([a-z0-9\\-]*))($|\\/))*$",
```

Comment:
Could we make this constrained string type for path ids another defined type and then just reference it?